_posts/2020-06-06-announcing-rustfest-2020.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Announcing RustFest Global/Online 2020 🎉"
+description: "Making RustFest Global for November 2020 in times of the Coronavirus"
+socialImageSrc: "/assets/posts/2020-global/2020-06-06_announcement.png"
+authors:
+  - dns2utf8
+---
+
+TLDR: The next RustFest will be on the weekend of ???2020-11-07 and 2020-11-08??? and it will be a online community conference 🎉
+
+# Future<Output = RustFest::Global2020>
+
+So how will it work?
+
+## CfP for talks
+
+The Call for Participation will be similar to the ones in the previous years, where we will start with a blind phase followed by a second round when a part of the team will see your names.
+
+Please note that dropping your name will not help your talk to make the cut.
```

Comment:
Doesn't it even disqualify? (not sure)

Reply:
It used to, but we have not finalized that yet.